File: 6_kyu/Count the smiley faces/solution.rb
def count_smileys(arr)
  smiley_regexp = /[:;][-~]*[)D]/
  smiley_arr = arr.select { |el| el.match(smiley_regexp) && (el.chars.length >= 2 && el.chars.length <= 3) }
  smiley_arr.length
end

File: 6_kyu/Count the smiley faces/solution.js
function countSmileys(arr) {
  const smileyRegexp = /[:;][-~]*[)D]/;
  const smileyArr = arr.filter(el => el.match(smileyRegexp) && (el.length >= 2 && el.length <= 3));
  return smileyArr.length;
}
